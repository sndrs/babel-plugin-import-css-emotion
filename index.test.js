const { transform } = require('babel-core');

const src = `
    import _ from 'lodash';
    import './index.test.css';

    const x = red;
`;

const { code } = transform(src, { plugins: './index.js' });

test('does what you would expect it to', () => {
    expect(code).toMatchSnapshot();
});
