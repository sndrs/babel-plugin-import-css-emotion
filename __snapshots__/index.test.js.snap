// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does what you would expect it to 1`] = `
"
import _ from 'lodash';
import { css, injectGlobal } from \\"emotion\\";const red = css\`
    color: red;


\`;injectGlobal\`

p {
    color: blue;
}


\`;

const x = red;"
`;
